app/javascript/app/pages/climate-goals/overview/timeline/timeline-component.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ReactMarkdown from 'react-markdown';
+import { TabletLandscape, TabletPortraitOnly } from 'components/responsive';
+import { Icon } from 'cw-components';
+import TimelineProvider from 'providers/timeline-provider';
+
+import issued from 'assets/icons/issued';
+import launched from 'assets/icons/launched';
+
+import styles from './timeline-styles.scss';
+
+const table = documents => (
+  <table className={styles.table}>
+    <tbody>
+      {documents.map(document => (
+        <tr key={document.label} className={styles.row}>
+          <td className={cx(styles.statusColumn, styles.cell)}>
+            <TabletLandscape>
+              <span className={styles.status}>
+                {document.year}
+              </span>
+            </TabletLandscape>
+            <Icon
+              icon={document.year === '2020' ? issued : launched}
```

Comment:
Can't we use the last one in case this changes? Or maybe it won't change 🤔

Reply:
what do you mean?

Reply:
I suppose that is a marked event. Nevermind then

Reply:
Hmm, status should be on the API side.

Reply:
it's that API: https://www.climatewatchdata.org/api/v1/timeline/IDN

Reply:
Yeah, I know. It's not there. Also, what about this API http://localhost:3000/api/v1/commitment_timeline_entries where should we use it?

Reply:
This is from the data they are providing in the data file. Maybe we can add status column there and ask them to fill it. Anyway, that could be done as a separate task because we have to ask them which data should we use.

Reply:
Yes, let's ask them. Or if it's only two statuses as it is now, we could have a flag `is_launched`.